build.sbt

```diff
@@ -4,6 +4,8 @@ name := "daffodil"
 
 organization in ThisBuild := "edu.illinois.ncsa"
 
+version in ThisBuild := "2.1.0-SNAPSHOT"
```

Comment:
I understand this is hardcoded now, but expected this naming convention would change. So are we going to continue with the NNN-SNAPSHOT convention for naming?

Reply:
The original plan was the change the version to something like "2.1.0-incubating-SNAPSHOT", only because other incubating projects seemed to do something similar. However, John pointed out that we don't actually need to have 'incubating' in the version--it's only required in the source tarball that is released. I think it makes things cleaner and easier to not have it (e.g. it doesn't inject "incubating" into the binary artifacts), and it's consistent with our old versioning scheme. For these reasons, i figured we just not include it in the version and make sure it's included when we do a release.

Reply:
The Branch and Tag Workflow wiki page has been updating to remove the "-incubating" in versions and tags/branches.